ui/design-system-components/src/components/Profile/Header/index.tsx

```diff
@@ -78,13 +78,14 @@ const Header: React.FC<HeaderProps> = ({
               <DidField
                 did={did.id}
                 isValid={validAddress}
-                copyLabel={copyLabel}
-                copiedLabel={copiedLabel}
+                {...(!copyLabel && !copiedLabel && { copiable: false })}
+                {...(!!copyLabel && { copyLabel: copyLabel })}
+                {...(!!copiedLabel && { copiedLabel: copiedLabel })}
```

Comment:
Yen, wouldn't this be more readable if you do:

```
copiable={copyLabel && copiedLabel}
copyLabel={copyLabel}
copiedLabel= {copiedLabel}
```

After all props can be nullable and I am not clear what the fancy object merge achieves other than taking more CPU compute resource :)

Reply:
lol :). Thanks for pointing it out Didd.